apps/Dataman/package.json

```diff
@@ -25,13 +27,13 @@
     "react-native-web": "~0.18.7"
   },
   "devDependencies": {
-    "eslint-config-custom": "*",
     "@babel/core": "^7.19.3",
     "@babel/preset-env": "^7.1.6",
     "@types/react": "~18.0.25",
     "@types/react-dom": "~18.0.9",
     "@types/react-native": "~0.70.5",
     "babel-preset-expo": "~9.2.1",
+    "eslint-config-custom": "workspace:*",
```

Comment:
@chandantiwari1 was there an issue caused by this change?

Reply:
No sir